etc/profile-a-l/feh.profile

```diff
@@ -7,30 +7,46 @@ include feh.local
 # Persistent global definitions
 include globals.local
 
+noblacklist ${HOME}/.config/feh
+
 include disable-common.inc
 include disable-devel.inc
 include disable-exec.inc
 include disable-interpreters.inc
+include disable-proc.inc
 include disable-programs.inc
 include disable-shell.inc
 
+whitelist /usr/share/feh
+whitelist /usr/share/yudit
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
```

Comment:
```suggestion
include whitelist-run-common.inc
include whitelist-runuser-common.inc
```

There can be image files in /usr/share (such as in /usr/share/icon and
/usr/share/doc) and maybe in /var/games, so I'd avoid whitelisting them.